homeassistant_cli/helper.py

```diff
@@ -99,7 +100,20 @@ def raw_format_output(
                 val = [match.value for match in fmtpair[1].find(item)]
                 row.append(", ".join(map(str, val)))
             result.append(row)
-
+        # Truncates data
```

Comment:
its crude and it somewhat works but I feel we should be able to do better than requiring users to pass in a width setting to get this.

How about using something like `shutil.get_terminal_size()` to get width and calculate a  default max_width based on columns/terminal_width with an option to ignore it ?

Reply:
then you could truly have a `auto` for column_width but should also have a -1 or `max` or something to have it just print everything as it does now.

Reply:
Thanks for the feedback. What about this:
```
                terminal_size = shutil.get_terminal_size()
                number_c = min([len(r) for r in result])
                columns_width = int(terminal_size.columns / number_c)
```

Reply:
looks about right - update the PR and we'll see.